### PR TITLE
Ccp 854

### PR DESCRIPTION
**Pull request by philipp-heyse**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Logs
```
Comment (sofib) [inline .gitignore]: we're usually ignoring package-lock.json

File: extension/addProperties.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const config = require('./config')
```
Comment (sofib) [inline extension/addProperties.js]: config is available via context.config

File: extension/package.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@shopgate/products-add-properties",
```
Comment (sofib) [inline extension/package.json]: Please check the results of `npm audit`

File: extension/test/unit/addProperties-test.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        value: 'added'
+      },
+      {
+        label: 'foo',
+        value: 'bar'
+      }
+    ])
+  })
+
+  it('should not do anything if no property is matched', async () => {
```
Comment (sofib) [inline extension/test/unit/addProperties-test.js]: maybe it should log configured properties that were not found as we may encounter mistypings

Comment (sofib) [inline extension/test/unit/addProperties-test.js]: this is case sensitive implementation, what do you think about case insensitive implementation? we could ask Daniela..

File: extension/addProperties.js (hunk 2 above)
Comment (philipp-heyse) [reply extension/addProperties.js]: yes, will change it

File: .gitignore (hunk 1 above)
Comment (philipp-heyse) [reply .gitignore]: adding it

File: extension/package.json (hunk 3 above)
Comment (philipp-heyse) [reply extension/package.json]: fixed. you do realize, this project has no actual (non-dev) dependencies?

File: extension/test/unit/addProperties-test.js (hunk 4 above)
Comment (philipp-heyse) [reply extension/test/unit/addProperties-test.js]: i disagree. not every product needs to have those properties. we'd be polluting the logs.

Comment (philipp-heyse) [reply extension/test/unit/addProperties-test.js]: makes sense, added

**Comment by philipp-heyse** [comment]
i don't see why we would create all those things for a 20 line extension without dependencies, but i'll do it on monday

File: extension/package.json (hunk 3 above)
Comment (sofib) [reply extension/package.json]: I do realise. When uploading to our system npm audit will be run and it makes no diff between type of a dependency.

**Pull request merged**